google/cloud/bigtable/internal/data_connection_impl_test.cc

```diff
@@ -1636,6 +1631,16 @@ TEST(DataConnectionTest, AsyncReadRowFailure) {
   EXPECT_THAT(resp, StatusIs(StatusCode::kPermissionDenied));
 }
 
+TEST(MakeDataConnection, DefaultsOptions) {
+  auto conn = bigtable::MakeDataConnection(
```

Comment:
> It makes an actual connection to the service

Might it be possible to move it to some place marked as an integration test?

Reply:
I want to check that the options are defaulted by `MakeDataConnection()` in a unit test.

I think my comment was inaccurate, and it reveals that I don't know what happens when a gRPC Stub is created. The test does not depend on the presence of a service/emulator i.e. it would pass even if I set the `EndpointOption` to some nonsense.

Reply:
Ack on the fishiness. I think the value added by the unit test is worth the fishiness.

> something that would at least stay on the same host?

I followed the example of [this test](https://github.com/googleapis/google-cloud-cpp/blob/df5eb3727a6a466d2073f7683766f89e1c4da3af/google/cloud/spanner/internal/spanner_stub_test.cc#L46). I think that is what you are suggesting?